apps/emqx_bridge_mqtt/src/emqx_bridge_mqtt.app.src

```diff
@@ -1,6 +1,6 @@
 {application, emqx_bridge_mqtt,
  [{description, "EMQ X Bridge to MQTT Broker"},
-  {vsn, "4.3.3"}, % strict semver, bump manually!
+  {vsn, "4.3.12"}, % strict semver, bump manually!
```

Comment:
no jumps.
App version is not the same as release version

Reply:
It looks like this bugfix will be included in the release of v4.3.13, should we increase one of the version numbers. e.g: `4.3.4`. And update the `appup.src`?